cime_config/machines/cmake_macros/intel_chrysalis.cmake

```diff
@@ -23,3 +23,7 @@ if (MPILIB STREQUAL impi)
   set(MPICXX "mpiicpc")
   set(MPIFC "mpiifort")
 endif()
+string(APPEND KOKKOS_OPTIONS " -DKokkos_ARCH_ZEN2=On")
```

Comment:
I think our next step (to keep code duplication under control) should be to include in the cmake mach files other cmake files for kokkos options, much like we do in EKAT.

Reply:
Agree.